src/main/java/oops/IS_A_Relation_Hierarchical.java

```diff
@@ -0,0 +1,49 @@
+package oops;
+class parent2{
+	public void p2assets() {
+		int car=2;
+		System.out.println("car "+car);
+	}
+}
+class chaild1 extends parent2{
+	@Override
+	public void p2assets() {
+		int car=2;
+		System.out.println("car "+car);
+	}
+
+	public void c1assets() {
+		int c1car=2;
+		System.out.println("c1car "+c1car);
+	}
+	
```

Comment:
Hi @varaprasadram1,  remove this empty space line for code looks good.
 regards,
 @yvpram.

Reply:
okay change now